pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2611,10 +2611,14 @@ protected void internalSetOffloadPolicies(AsyncResponse asyncResponse, OffloadPo
             }
 
             policies.offload_policies = offloadPolicies;
+            String updatedOffloadPolicies = jsonMapper().writeValueAsString(policies.offload_policies);
             globalZk().setData(path, jsonMapper().writeValueAsBytes(policies), nodeStat.getVersion(),
```

Comment:
```suggestion
            globalZk().setData(path, updatedOffloadPolicies, nodeStat.getVersion(),
```

Reply:
Thanks, the variable `updatedOffloadPolicies` here is just used to print logs and `byte [] policies` is written to znode.